### PR TITLE
Added archived field to Task model

### LINKED ISSUE #1006: Add the ability to archive a task
# Problem

We need to add the ability to archive a task.

- Add an `archived` column to tasks

### PR DESCRIPTION
**Pull request by vishaldeepak**
# What's in this PR?
Added Archived field to Tasks

Fixes #1006

### CODE REVIEW

**Comment by joshsmith** [comment]
Hey @vishaldeepak I know this wasn't in the task, but can you add this to the `TaskView` and `TaskViewTest` so we can be sure it's emitted to the front-end? Make sure they're alpha-ordered.

File: lib/code_corps/model/task.ex [hunk 1]
```diff
@@ -63,7 +64,7 @@ defmodule CodeCorps.Task do
   def update_changeset(struct, params) do
     struct
     |> changeset(params)
-    |> cast(params, [:status])
+    |> cast(params, [:status, :archived])
```
Comment (joshsmith) [inline lib/code_corps/model/task.ex]: Can you reorder this to be in alpha order?

File: test/lib/code_corps/model/task_test.exs [hunk 2]
```diff
@@ -70,5 +70,13 @@ defmodule CodeCorps.TaskTest do
...
       refute closed_at
     end
+
+    test "archived field changes appropriately" do
+      changes = Map.put(@valid_attrs, :archived, true)
+      changeset = Task.update_changeset(%Task{task_list_id: 1}, changes)
+      %{archived: archived} = changeset.changes
+      assert changeset.valid?
+      assert archived
+    end
```
Comment (joshsmith) [inline test/lib/code_corps/model/task_test.exs:80]: Solid test here. 👍

**Review by joshsmith — CHANGES_REQUESTED** [review]
Quick change, and see comment above.

**Comment by vishaldeepak** [comment]
Made changes accordingly

**Review by joshsmith — DISMISSED** [review]

**Review by joshsmith — DISMISSED** [review]

**Review by joshsmith — APPROVED** [review]

**Pull request merged**